pythoncz/templates/_communication.html

```diff
@@ -82,4 +82,12 @@
             </span>
         </a>
     </li>
+    <li class="communication-item">
+        <a href="https://discord.gg/yUbgArVAyF">
+            <span class="communication-item-icon"><i class="fab fa-discord"></i></span>
+            <span class="communication-item-caption">
+                Python CZ/SK
```

Comment:
Do you think it would make sense to change this to "Python CZ/SK chat" for those who won't recognize the Discord logo? Or, perhaps, something like this:

```diff
- {% if lang == 'cs' %}Organizační Slack{% else %}Slack for organizers{% endif %}
+ {% if lang == 'cs' %}Chat pro organizátory{% else %}Chat for organizers{% endif %}
```

and

```diff
- Python CZ/SK
+ Komunitní chat
```

...so it's clearer what is for what? I'm open to opinions, take my comment merely as a suggestion 🙂

Reply:
And personally, I'd sort it like this:

1. Pyonýři (IMHO most mainstream)
2. Discord (shiny new stuff)
3. e-mail groups (still surviving)
4. twitter groups
5. irc channels (almost dead? IDK)
6. slack (invite-only, for orgs/admins/mods, not interesting that much for regular community members)

Reply:
Originally I had `Python CZ/SK Discord` there, but visually it seemed to take bit too much space in the communications block.

My thoughts about this:

One would pass/ignore the icon if:
- he doesn't care about new stuff
- knows but doesn't like Discord

Otherwise interested if:
- wants to see and try new stuff 
- knows what it means (and has discord account)

So, in some sense I would vouch for something along the lines of `Komunitní Discord` for cz and `Python  CZ/SK`.

Reply:
What about just “Discord”?
These are all CZ/SK Python community chats/forums; if that bit is unclear we should label the entire category.
```suggestion
                Discord
```

Reply:
I don't really mind, `Discord` works too.